homework-assignments-1/task-4.ts

```diff
@@ -0,0 +1,154 @@
+const ShippingCost: Record<string, number> = {
+    Tashkent: 10.8,
+    London: 15.7,
+    Toronto: 11.3,
+    Sydney: 14.7
+} as const
+
+const userLoginInfo: Record<string, string> = {
+    "user@gmail.com": "user123"
+} as const
+
+class Product {
+    constructor(public name: string, public price: number, public stock: number) { }
```

Comment:
It is not good idea to expose class member properties as public. instead, create getters and setters in order to interact with them
getName
getPrice
getStock